app/forms/TextFormProvider.scala

```diff
@@ -17,22 +17,22 @@
 package forms
 
 import forms.mappings.Mappings
-import play.api.data.{Form, Mapping}
+import play.api.data.Form
 import uk.gov.hmrc.domain.Nino
 
 import javax.inject.Inject
 
-class TextFormProvider @Inject()() extends Mappings {
+class TextFormProvider @Inject()() {
 
-  val nameRegex = "^[a-zA-Z\\-' ]+$"
-  val textAreaRegex = """^[a-zA-Z0-9\-'" \t\r\n,.@/]+$"""
-  val textAreaMaxLength = 160
+  protected[forms] val nameRegex = "^[a-zA-Z\\-' ]+$"
```

Comment:
Do we need `[forms]` here? As it is already directly in forms package?

Reply:
it's mainly so the test in the same package can access the regex to assert and also stops consuming classes from accessing them